client-base/src/main/webapp/html/javascript/utils.js

```diff
@@ -155,6 +155,7 @@ function bracketsAreBalanced(str) {
  */
 function matchCounter() {
     var keyword = $('#keywords').val();
+
```

Comment:
Going to accept these changes, but unfortunately, this file has become a monster of 100lines before the arrival of @manlikleo. As next task, let's do at least the basic refactoring of splitting it into included files, based on the functionality categories.